src/components/verifyOtp/VerifyOtp.js

```diff
@@ -0,0 +1,15 @@
+import React from "react";
+
+import InputControl from "components/InputControl/InputControl";
+
+import styles from "./VerifyOtp.module.scss";
+
+function VerifyOtp() {
+  return (
+    <div className="verifyOtp">
+      <InputControl label="OTP" placeholder="Enter OTP" />
+    </div>
```

Comment:
where is the button of this component? Ideally any component should be independent as much as we can.

all the functionalities of sending the otp to receiving the otp must be in this component and if needed to contact with register/login component contact with the help of props